### PR TITLE
adding twirf #7

### PR DESCRIPTION
**Pull request by gnunicorn**
- Speaker announcement
  - ViewSource Collaboration announcement
  - mini update on tickets (1st week of Aug)

I am tired and can't review it properly anymore. Input highly appreciated!

Rendered:
screen shot 2016-07-22 at 18 34 27-fullpage

### CODE REVIEW

File: _posts/2016-07-22-this-week-in-rustfest-7-first-talks-and-viewsource.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "This Week in RustFest 7: first talks announced, collaborating with viewsource"
+socialImageSrc: '/assets/posts/twirf-7-speakers.png'
+---
+
+## First talks up
+
+As already mentioned last week, we've closed the CfP and by now informed everyone about their participation. At the moment we are still in process to gathering all details and ensuring the logistics for everyone, but the basic talk schedule is clear now. Which allows us to announce the first set of  confirmed speakers and their [talks](/talks/): 🎉
```
Comment (badboy) [inline _posts/2016-07-22-this-week-in-rustfest-7-first-talks-and-viewsource.md]: "still in process **of** gathering"

File: _posts/2016-07-22-this-week-in-rustfest-7-first-talks-and-viewsource.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+</div>
+
+
+(From right to left) **Stephan Hügel** will be sharing with us how he taught [himself Rust through the medium of coordinate system transforms and FFI](/talks/#teaching-myself-rust-through-the-medium-of-coordinate-system-transforms-and-ffi), while **Sonja Heinen** takes us a little off the beating past and asks us to ponder with her about [Hobby-oriented Programming](/talks/#hobby-oriented-programming). When porting other projects to Rust, **Martin Hellspong** explains us [Why we're better off using QuickCheck](/talks/#why-you-d-be-better-off-using-quickcheck-when-porting-to-rust) and if you are coming from a different language,  **Brian Pearce** advises you to start [Learning to Love the Unfamiliar](/talks/#learning-to-love-the-unfamiliar). Lastly for this week, **Tim Cameron Ryan** takes us into the depth of programming Internet of Things programming with [Sensors, Servos, and Signals with Rust](/talks/#sensors-servos-and-signals-with-rust)
```
Comment (badboy) [inline _posts/2016-07-22-this-week-in-rustfest-7-first-talks-and-viewsource.md]: one "programming" to many
Maybe

> takes us into the depth of programming the Internet of Things with ..

File: _posts/2016-07-22-this-week-in-rustfest-7-first-talks-and-viewsource.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+
+We will complete this list over the next week or so. So keep an eye out for updates of our [talks section on the website](/talks/). And of course we'll do a round up every week here, too.
+
+
+## Collaborating with View Source
+
+We are super please to announced we'll be collaborating with the awesome View Source Conference, which takes place earlier the same week as RustFest – 12-14th of September. Check it out here:
```
Comment (badboy) [inline _posts/2016-07-22-this-week-in-rustfest-7-first-talks-and-viewsource.md]: → We are super pleased to announce ...

d moved to `please`, removed from `announced`

**Comment by badboy** [comment]
Tiny little remarks, otherwise good to go

**Comment by gnunicorn** [comment]
sorry. got distracted watching terrible trailers on youtube. merging now.

**Comment by badboy** [comment]
Sounds like fun, care to share?^^

**Comment by gnunicorn** [comment]
@badboy for sure:
- https://www.youtube.com/watch?v=_jM4lLYhjQs
- https://www.youtube.com/watch?v=jxKyDxthvOI
- https://www.youtube.com/watch?v=AfuVgfuaTbQ
- https://www.youtube.com/watch?v=W61-oe2g_qM
- https://www.youtube.com/watch?v=t0hSzMVVKC8

and for some reason:
- https://www.youtube.com/watch?v=s_BzTJ7etkA

**Comment by gnunicorn** [comment]
I started the first because of [review I came across when looking for the 301 moved](https://thatdutchfilmlover.wordpress.com/2014/02/09/nurse-3d-2013/) –  one that made it sound super terrible. But it said "gore" and I was like ... I could do some gore...

**Pull request merged**